### PR TITLE
Eliminate script snapshot support

### PR DESCRIPTION
**Pull request by cbracken**
Script snapshots were only every used in Dart 1 mode, which is no longer
supported.

### CODE REVIEW

**Review by goderbauer — APPROVED** [review]
LGTM

File: packages/flutter_tools/lib/src/base/build.dart [hunk 1]
```diff
@@ -67,69 +67,6 @@ class GenSnapshot {
   }
 }
 
-/// Dart snapshot builder.
```
Comment (tvolkert) [inline packages/flutter_tools/lib/src/base/build.dart]: TIL that the `--url_mapping` flags also do nothing in a Dart 2 world.  /cc @a-siva

**Review by tvolkert — APPROVED** [review]
LGTM

**Review by sbaranov — APPROVED** [review]

**Comment by cbracken** [comment]
One in a series of fixes for #17051.

**Pull request merged**